index-singlecell.md

```diff
@@ -52,10 +54,43 @@ In this section we list all tools that have been integrated in the the single ce
+|  ScanPy | AnnData CSV LOOM excel mtx umi-tools | Y         | CPM                                       | Cell Cycle ComBat MNNCorrect BBKNN | PCA leiden louvain       | PCA tSNE UMAP                       | dendrogram PAGA    | Y                       | one of the first with 10X support                                               |
+|  Scater | SCE SC3                              | Y         | CPM FPKM TPM                              |                                    | PCA                      | PCA tSNE UMAP PHATE diffmap         |                    |                         |                                                                                 |
+|  Seurat | CSV 10X H5 Alevin                    | Y         | LogNormalise Centered-log Relative-counts | SCTransform SNNAnchor              | SharedNN                 | PCA tSNE UMAP                       |                    | Y                       |                                                                                 |
+|  RaceID | CSV SCSeq                            | Y         | basic library size                        | Linear CCcorrect (Cell cycle)      | k-means k-medians hclust | PCA tSNE UMAP Fruchterman-Rheingold | StemID FateID      |                         |                                                                                 |
+| Monocle |                                      | Y         | basic library size                        | Y                                  | Louvain KNN GMM          | ICA tSNE L1-graph SimplePPT DDRTree | Y                  | Y                       | Works well with CellRanger                                                      |
+|  DESeq2 | CSV htseq-count summarizedexperiment | Y         | DESeq2                                    |                                    | Y                        |                                     |                    |                         |                                                                                 |
+|  scPipe |                                      | Y         | Y                                         | Y                                  | Y                        | Y                                   |                    |                         | CelSeq2, etc. Uses Scater and SCRAN for downstream. Also perform pre-processing |
+
+
+
 Tool | Description | Reference
 --- | --- | ---
 {% include tool.html id="FastQC" %} | A quality control tool for high throughput sequence data | -
-{% include tool.html id="MultiQC" %} | MultiQC aggregate results from bioinformatics analyses into a single report | [Ewels et al. 2016](https://doi.org/10.1093/bioinformatics/btw354){:target="_blank"}
+ | MQC | [Ewels et al. 2016](https://doi.org/10.1093/bioinformatics/btw354){:target="_blank"}
```

Comment:
why this change?

Reply:
It's used in Line62, so I was probably in the process of moving it.